### ISSUE TITLE
[New Rule] Multiple isset statements SHOULD be combined into one

### ISSUE THREAD

**Issue by lenaorobei** [magento/magento-coding-standard#27]
### Rule
Multiple `isset` statements SHOULD be combined into one.

### Reason

Just the code style question aimed to make the codebase consistent.

**Bad example:**
```php
if (isset($a) && isset($b) && isset($c)) {
    // code goes here
}
```

**Good example:**
```php
if (isset($a, $b, $c)) {
    // code goes here
}
```
### Implementation
Check for multiple `isset` statements combined with `&&` inside one condition.

**Comment by buskamuza** [comment]
As for me, it's OK to have both styles. There can be use cases with `if (isset($a) && empty($b))`, and in this case it looks more consistent with the bad example.
If many people vote that this sniff can improve readability of the code, let it be. Otherwise I would not restrict this.

**Comment by ravmenon** [comment]
For extensions, this should be only a low severity (style) warning as the first idiom is very common.

**Comment by lenaorobei** [comment]
No updated and votes here, so both styles are ok.
No need to enforce.
